hypervisor/src/mshv/mod.rs

```diff
@@ -53,6 +53,12 @@ pub use {
 
 pub const PAGE_SHIFT: usize = 12;
 
+// These constants are derived from GHCB spec page 26.
```

Comment:
It is better to referenced the chapter name than a page number.

Reply:
Fixed